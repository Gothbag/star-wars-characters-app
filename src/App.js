import React from "react";

import "./App.css";
import Home from "./containers/Home";

class App extends React.PureComponent {
    render() {
        return (<section className='container' style={{height: '100%'}}>
            <section className='row center' style={{height: '100%'}}>
                <Home/>
            </section>
        </section>);
    }
}

export default App;
