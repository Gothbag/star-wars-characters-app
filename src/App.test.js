import React from 'react';
import App from './App';

it('renders without crashing', () => {
    const wrapper = shallow(<App/>);
    expect(wrapper.find("section.container").length).toBe(1);
});
