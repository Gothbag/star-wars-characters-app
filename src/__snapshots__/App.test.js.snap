// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <section
        className="row center"
        style={
                Object {
                        "height": "100%",
                      }
        }
>
        <Connect(Home) />
</section>,
      "className": "container",
      "style": Object {
        "height": "100%",
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Connect(Home) />,
        "className": "row center",
        "style": Object {
          "height": "100%",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "section",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <section
          className="row center"
          style={
                    Object {
                              "height": "100%",
                            }
          }
>
          <Connect(Home) />
</section>,
        "className": "container",
        "style": Object {
          "height": "100%",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Connect(Home) />,
          "className": "row center",
          "style": Object {
            "height": "100%",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "section",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
